src/server/update-member-expiry-date.ts

```diff
@@ -2,9 +2,8 @@ import { db } from '@/db';
 import { memberTable } from '@/db/schema';
 import { eq } from 'drizzle-orm';
 
-export const updateMemberExpiryDate = async (id: string, idType: 'clerkId' | 'id') => {
-    const now = new Date();
-    const expiryDate = new Date(`${now.getFullYear() + 1}-01-01`);
+export const updateMemberExpiryDate = async (id: string, idType: 'keycloakId' | 'id') => {
+    const expiryDate = new Date(`2026-01-01`);
```

Comment:
actually a lot of ppl asked about this. Can we change the expired date to actually one year from the time they registered, or say smth when they pay.

Reply:
> say smth when they pay.

What do you mean by this? If you mean making it more clear the membership will expire on the 1st of January, then we can do that. This is probably suited for another PR where we change the onboarding flow.

We won't make the expiry date exactly one year from membership purchase as that complicates things logistically such as committee handovers and making significant changes to the membership for the following year.

Reply:
yea I mean if we make the expired date at the end of that year, we should mention smth like "Your membership will be expired at xxxx" when they pay.